packages/algob/src/types-input.ts

```diff
@@ -0,0 +1,35 @@
+import * as z from 'zod';
+
+export const AddressSchema = z.string();
```

Comment:
Do we need this?

Reply:
This is how address is decoded. 
It's used four times in the ASA definition:
```
  manager: AddressSchema.optional(),
  reserve: AddressSchema.optional(),
  freeze: AddressSchema.optional(),
  clawback: AddressSchema.optional()
```

Reply:
OK, thanks.